src/main/assembly/dist/bag/metadata/amd/v1.0/amd.xsd

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2012 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified"
+    targetNamespace="http://easy.dans.knaw.nl/easy/dataset-administrative-metadata/"
+    xmlns:damd="http://easy.dans.knaw.nl/easy/dataset-administrative-metadata/" xmlns:wfs="http://easy.dans.knaw.nl/easy/workflow/">
+    <xs:import namespace="http://easy.dans.knaw.nl/easy/workflow/" schemaLocation="wfs.xsd"/>
+    <!-- import the versioned schema of wfs -->
+    <xs:annotation>
+        <xs:documentation>Schema to describe the administrative metadata for datasets in EASY, v1.0.
+            Changes from v0.1 to v1.0:
+            * delete: groupIds in the amd
+            * delete: datasetState/previousState no longer recorded, as state is not a property to be archived
+            * delete: stateChangeDates
+            * add: permissionGranterId (TODO: better name)
```

Comment:
suggestion: `datasetOwnerId`? Its a bit more general, but probably we could do more with it than just use it for permission granting...

Sidenote: isn't this always the same value as we have in `bag-info.txt` for the owner?

Reply:
i guess for a lot of cases, the two are identical.
however there are use cases where they are not, and at the moment that's a problem with a work-around

Reply:
intellectualPropertyRightsHolder ?

Reply:
sounds good. alternatively _intellectualPropertyHolder_ (without the _Rights_)

Reply:
AccessGranter ?